pxr/base/tf/preprocessorUtils.h

```diff
@@ -63,7 +63,7 @@ ARCH_PRAGMA_MACRO_TOO_FEW_ARGUMENTS
 ///
 /// \ingroup group_tf_Preprocessor
 /// \hideinitializer
-#if defined(ARCH_COMPILER_MSVC)
+#if defined(ARCH_COMPILER_MSVC) && (!defined(_MSVC_TRADITIONAL) || _MSVC_TRADITIONAL)
```

Comment:
If your open to the suggestion, you might want to just create a `ARCH_MSVC_TRADITIONAL_PREPROCESSOR` macro equivalent to 
`defined(ARCH_COMPILER_MSVC) && (!defined(_MSVC_TRADITIONAL) || _MSVC_TRADITIONAL)` that can be referenced in `preprocessorUtils.h` and `preprocessorUtilsLite.h`.

Support for the conformant preprocessor is exciting to see.

Reply:
Can do that - will still be blocked by the warnings emitted from the MSVC include chain. As noted above, I do not see them locally, as I am compiling them against a newer version. With that update, it compiles cleanly "on my machine.com"

Reply:
> Microsoft Blog: Windows 10 SDK version 2104 has been released, which includes the changes needed for C11 and C17 as well as the conformant preprocessor
This seems to match with Visual Studio 2019.
I will update the PR with the asked-for change (since I keep them as patch local), but then this should either be closed or left unresolved until an SDK update.